modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/common/beans/WindowsPluginProperties.java

```diff
@@ -23,61 +23,70 @@
  */
 public class WindowsPluginProperties {
 
-	private String keyStorePassword;
-	private String privateKeyPassword;
-	private String commonName;
-	private String authPolicy;
-	private int notBeforeDays;
-	private int notAfterDays;
+    private String keyStorePassword;
+    private String privateKeyPassword;
+    private String commonName;
+    private String authPolicy;
+    private String domain;
+    private int notBeforeDays;
+    private int notAfterDays;
```

Comment:
Also since this object is meant to be reading values from a configuration file and just keep one-time values, isn't it a good idea to make this class immutable, so that value assignment happens one time at the constructor level and we expose only getter methods as public, just to read the values.